build-system/test-configs/karma.conf.js

```diff
@@ -26,8 +26,8 @@ const COMMON_CHROME_FLAGS = [
   // Allows simulating user actions (e.g unmute) which otherwise will be denied.
   '--autoplay-policy=no-user-gesture-required',
   // Makes debugging easy by auto-opening devtools.
-  argv.debug ? '--auto-open-devtools-for-tabs' : null,
-].filter(Boolean);
+  argv.debug ? '--auto-open-devtools-for-tabs' : '',
+];
```

Comment:
This inserts a blank string entry to the array while `.filter(Boolean)` removes it. Any other way to fix this?

Reply:
My previous solution prevented this. I preferred that approach but @danielrozenberg didn't like the idea of pushing to a `const` list.
`
if (argv.debug) {
  COMMON_CHROME_FLAGS.push('--auto-open-devtools-for-tabs');
}
`

Perhaps we could build the list in a function?

`
function getCommongChromeFlags() {
  let flags = [
    // Dramatically speeds up iframe creation time.
    '--disable-extensions',
    // Allows simulating user actions (e.g unmute) which otherwise will be denied.
    '--autoplay-policy=no-user-gesture-required',
  ];
  if (argv.debug) {
    flags.push('--auto-open-devtools-for-tabs');
  }
  return flags;
}

const COMMON_CHROME_FLAGS = getCommongChromeFlags();
`

Reply:
`karma.conf.js` is meant to be a collection of mostly statically determined initial config, so I don't think we should add a builder function to this file. (The more complicated dynamic config building is done by the `RuntimeTestConfig` class in `runtime-test-base.js`.)

To keep this simple, let's use the first option you listed here. (Here's a [discussion](https://stackoverflow.com/questions/23436437/why-can-i-change-value-of-a-constant-in-javascript) on pushing a value to a `const`.)

Reply:
Thanks for the discussion link,  that's always been my perspective on const Objects in JS but it's good to hear it reiterated.